templates

```diff
@@ -2,9 +2,9 @@ Template: siptreadmill/authentication/provider
 Type: select
 Choices: github, att
 Translations: Github, AT&T APIMatrix
-Default: sendmail
-Description: Which authentication provider do you want to use for _APP_NAME_?
- Chose the authentication provider you have setup _APP_NAME_.
+Default: github
+Description: Which authentication provider do you want to use for $APP_NAME?
+ Chose the authentication provider you have setup $APP_NAME.
```

Comment:
Why did you change this? AFAIK it's supposed to be `_APP_NAME_` and the script replaces it.